src/content/whats-new/2024/02/whats-new-02-27-compute-add-on-sku

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Launching the Compute Add On SKU'
+summary: 'Unlock access to all new capabilities, including Live Archives and IAST, with a single SKU'
```

Comment:
@KarlTheFogNR customers who want to have live archives also need the "Live Archives Storage" SKU, right?

Reply:
@rhetoric101 This is true, but this is not a What's New post about Live Archives or IAST and what they require so much as an explainer of the Add On SKU itself. I'd prefer to keep it simple.